golang/cosmos/app/app.go

```diff
@@ -934,8 +934,10 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
 			// Each CoreProposalStep runs sequentially, and can be constructed from
 			// one or more modules executing in parallel within the step.
 			CoreProposalSteps = []vm.CoreProposalStep{
-				// Upgrade ZCF only
-				vm.CoreProposalStepForModules("@agoric/builders/scripts/vats/upgrade-zcf.js"),
+				// Upgrade Zoe + ZCF
+				vm.CoreProposalStepForModules("@agoric/builders/scripts/vats/replace-zoe.js"),
+				// Revive KREAd characters
+				vm.CoreProposalStepForModules("@agoric/builders/scripts/vats/revive-kread.js"),
```

Comment:
I'm not privy to the plan so I'm just assuming this is supposed to be done

Reply:
Yes, Zoe needs to be updated, and that forces us to repair KREAd